scripts/prepare_qodana.sh

```
@@ -0,0 +1,10 @@
+#! /bin/sh
+
+sudo apt update
+sudo apt install -y qt6-base-dev
+
+mkdir -p build
+cd build
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)